wiki/osu!academy/id.md

```diff
@@ -1,63 +1,57 @@
+
+Kami berharap pemain baru dapat menemukan video osu!academy ini informatif, dan akan membantu meringankan hal yang harus dipelajari dari game ini. 
+Jika Anda memiliki pertanyaan atau komentar, Anda bisa mengirimnya kepada [ztrot](/users/6347), dia adalah pemilik satu-satunya osu!academy bersamaan dengan kanal YouTube-nya.
+
+[Berikut merupakan trailernya](https://www.youtube.com/watch?v=z5gy34k3RI0&feature=c4-overview&list=UUMeRgqzTfC5ja40B6kM6pdg).
+
+## Daftar Episode
+
+| Thumbnail Episode | Tautan | Deskripsi |
+|----------------------------------------------------------|--------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ![](img/Academy_1.png) | [Episode 1](https://www.youtube.com/watch?v=cz522ZAs5aQ "Episode 1") | Dalam episode ini, kita akan membahas cara penginstalan dan pembuatan akun! |
+| ![](img/Academy_2.png) | [Episode 2](https://www.youtube.com/watch?v=mswLEXK0eDk "Episode 2") | Dalam episode ini, kita akan membahas gameplay standard dan bagaimana mengubah pengaturan gaya bermain! |
+| ![](img/Academy_3.png) | [Episode 3](https://www.youtube.com/watch?v=UAomychlbic "Episode 3") | Dalam episode ini, kita akan membahas mode permainan osu!mania bersama LoliFlan! |
+| ![](img/Academy_4.png) | [Episode 4](https://www.youtube.com/watch?v=6WKZE2HPOK8 "Episode 4") | Dalam episode ini, kita akan membahas mode permainan CtB |
+| ![](img/Academy_5.png) | [Episode 5](https://www.youtube.com/watch?v=f_uSO2ESCRI "Episode 5") | Dalam episode ini, kita akan membahas mode permainan Taiko bersama Tasha! |
+| ![](img/Academy_6.png) | [Episode 6](https://www.youtube.com/watch?v=cyYRl-a5xII "Episode 6") | Dalam episode ini, kita akan membahas mode multiplayer! |
+| ![](img/Academy_7-0.png) | [Episode 7-0](https://www.youtube.com/watch?v=WKS8Zhut9XU "Episode 7-0") | Dalam episode ini, kita akan mulai melangkah ke dunia pembuatan beatmap! |
+| ![](img/Academy_7-1.png) | [Episode 7-1](https://www.youtube.com/watch?v=RKLanv4pvJc "Episode 7-1") | Dalam episode ini, kita akan membahas dasar-dasar pembuatan slider! |
+| ![](img/Academy_7-2.png) | [Episode 7-2](https://www.youtube.com/watch?v=8nsbrOhLE9w "Episode 7-2") | Dalam episode ini, kita akan membahas bagaimana mengatur timing beatmap Anda! |
+| ![](img/Academy_7-3.png) | [Episode 7-3](https://www.youtube.com/watch?v=MhIuPvQjLbU "Episode 7-3") | Dalam episode ini, kita akan membahas grid dan apa fungsi mereka! |
+| ![](img/Academy_8.png) | [Episode 8](https://www.youtube.com/watch?v=PFEYlQfiJHQ "Episode 8") | Dalam episode ini, kita akan membahas bagian penting dari mapping yang dapat membuat map lebih baik atau buruk: pemberian hitsound! |
+| ![](img/Academy_9.png) | [Episode 9](https://www.youtube.com/watch?v=bTGBspoMFVA "Episode 9") | Dalam episode ini, kita akan membahas modding beatmap! |
```

Comment:
`membahas modding beatmap!` add `tentang` when the object is verb

> mebahas tentang modding beatmap

will effect others lines